.circleci/config.yml

```diff
@@ -1,5 +1,181 @@
 version: 2.1
 
+orbs:
+  # Using inline orb for now
+  getting-started-smoke-test:
+    orbs:
+      macos: circleci/macos@2
+      android: circleci/android@2.0.3
+      flutter-orb: circleci/flutter@1.0.1
+
+    executors:
+      mac-executor:
+        macos:
+          xcode: 13.3.1
```

Comment:
Can we make this `13.2` since that will be our minimum version on our next release?